scripts/convert.py

```diff
@@ -19,6 +19,11 @@
 
 from .extract import Plugins as Extractor
 
+import numpy as np
+import face_recognition
+from PIL import Image
```

Comment:
PIL and glob are not current requirements but the tasks they perform are replicated elsewhere in the FS toolchain. consider consolidating to one choice

Reply:
removed imports not in requirements